package.nuspec

```diff
@@ -9,8 +9,23 @@
         <repository type="git" url="$repoUri$" commit="$commitId$"/>
     </metadata>
     <files>
-        <file src="packages/**/*.*" />
-        <file src="scripts/**/*.*" />
-        <file src="*.*" />
+      <!-- iOS device debug -->
+      <file src="DerivedData\polyester\Build\Products\Debug-iphoneos\libFRN*.a" target="Debug-iphoneos"/>
+      <!-- iOS device ship -->
+      <file src="DerivedData\polyester\Build\Products\Release-iphoneos\libFRN*.a" target="Ship-iphoneos"/>
+      <!-- iOS simulator debug -->
+      <file src="DerivedData\polyester\Build\Products\Debug-iphonesimulator\libFRN*.a" target="Debug-iphonesimulator"/>
+      <!-- iOS simulator ship -->
+      <file src="DerivedData\polyester\Build\Products\Release-iphonesimulator\libFRN*.a" target="Ship-iphonesimulator"/>
+      <!-- macOS debug -->
+      <file src="\DerivedData\polyester\Build\Products\Debug\libFRN*.a" target="Debug-macosx"/>
+      <!-- macOS ship -->
+      <file src="DerivedData\polyester\Build\Products\Release\libFRN*.a" target="Ship-macosx"/>
```

Comment:
Can we have an explicit list instead of using wildcards. Being explicit with a manifest has in practice been no inconvenience in practice and great at making changes intentional

Reply:
Done